views/js/runner/plugins/tools/itemThemeSwitcher/itemThemeSwitcher.js

```diff
@@ -111,14 +122,15 @@ define([
                 title: __('Change the current color preset'),
                 icon: 'contrast',
                 text: __('Contrast')
-            });
-
-            this.menuButton.on('click', function (e) {
+            }).on('click', function (e) {
                 e.preventDefault();
                 testRunner.trigger('tool-themeswitcher-toggle');
+            }).on('openmenu', function() {
+                testRunner.trigger('plugin-open.' + pluginName, state.selectedTheme);
```

Comment:
Why don't you use the method trigger on the plugin instance itself  ?
```
self.trigger('open', state.selectedTheme);
```

https://github.com/oat-sa/tao-core/blob/master/views/js/core/plugin.js#L199

Reply:
I firstly used the plugin trigger method, however it adds extra parameter that pollutes the arguments, and breaks the compatibility with regular events in the context of probes.

plugin trigger:
```
event(plugin, ...)
```

testRunner trigger:
```
event(...)
```